src/systems/filecoin_nodes/clock/clock_subsystem.go

```diff
@@ -2,10 +2,27 @@ package clock
 
 import "time"
 
-func (_ *WallClock_I) NowUTC() Time {
+func (_ *Clock_I) UTCTime() Time {
 	return Time(time.Now().Unix())
 }
 
-func (_ *WallClock_I) NowUTCNano() Time {
+func (_ *Clock_I) UnixNano() Time {
 	return Time(time.Now().UnixNano())
 }
+
+func (self *Clock_I) CurrentEpoch() ChainEpoch {
+	return self.currentEpoch_
+}
+
+func (self *Clock_I) CurrentEpochState() ChainEpoch {
+	panic("")
+	// if self.UnixNano()-self.currentEpochStart > self.cutoffAt_ {
+	// 	return ChainEpochState.PastCutoff
+	// }
+	// return ChainEpochState.Active
+}
+
+func (self *Clock_I) ResetEpoch(genesisTime Time) struct{} {
+	panic("")
+	// self.currentEpoch = (self.UnixNano() - genesisTime) / epochDuration
+}
```

Comment:
remove

Reply:
Was thinking this could be useful when you are syncing with network again and want to remove skew